app/app/AnswerRepository.php

```diff
@@ -1,298 +0,0 @@
-                    foreach ($question_ids as $question_id) {
-                        $rating = 0;
-                        $question_id_key = '' . $question_id;
-                        if (isset($ratings[$question_id_key])) {
-                            $rating = $ratings[$question_id_key];
-                        }
-                        $totalRating += $rating;
-                    }
-                    $location->rating = $totalRating / $num_questions;
-                }
-            } else {
-                $ratingSystem = 'universal';
-            }
-        }
-        if ($ratingSystem === 'universal') {
-            foreach ($locations as $location) {
-                $location->rating = $location->getAccessibilityRating('universal');
-            }
-        }
-    }
-
-    private static function getAnswersForLocation(string $location_id)
-    {
-        if (!Session::has('answers_'.$location_id)) {
-            Session::put('answers_'.$location_id, []);
-        }
-        $answers = Session::get('answers_'.$location_id);
-        if (!array_key_exists($location_id, $answers)) {
-            $answers = [];
-        }
-
-        Session::put('answers_'.$location_id, $answers);
-
-        return $answers;
-    }
-
-    public static function saveAnswer(string $location_id, int $question_id, int $answer_value)
-    {
-        /*Validate that the corresponding question doesn't have "is_always_required" set when answer_value indicates
-        that the question's feature is not required.
-        For example, "Lit parking lot" is required regardless of the specific location because no matter what the
-        location is, if you use a wheelchair and have poor eyesight, you'd need struggle to get there.
-        The client isn't allowed to indicate these few questions as "not required" because it is required for
-        EVERY location.*/
-        if ($answer_value === 2) {
-            $question = DB::table('question')
-                ->where('id', '=', $question_id)
-                ->where('is_always_required', '=', '0')
-                ->first();
-
-            if (!$question) {
-                throw new Exception('Unable to find question with id '.$question_id.' that is not always required');
-            }
-        }
-        Session::put('answers_'.$location_id.'_'.$question_id, $answer_value);
-    }
-
-    public static function removeAnswer(string $location_id, int $question_id)
-    {
-        Session::forget('answers_'.$location_id.'_'.$question_id);
-    }
-
-    public static function getAnswer(string $location_id, int $question_id)
-    {
-        $key = 'answers_'.$location_id.'_'.$question_id;
-        if (!Session::has($key)) {
-            return '';
-        }
-        return Session::get($key);
-    }
-
-    public static function saveComment(string $location_id, int $question_category_id, string $comment)
-    {
-        $key = 'answers_'.$location_id.'_comment_'.$question_category_id;
```

Comment:
@passionInfinite can you find a way to use a PHPUnit test to test a scenario where you sign in using Laravel's auth?  This should help us keep test coverage of functionality available only after signing in.

This is what I came across but I didn't experiment with it:
https://laravel.com/docs/5.2/testing#sessions-and-authentication